src/Microsoft.Extensions.CommandLineUtils/CommandLine/CommandLineApplication.cs

```diff
@@ -43,6 +44,8 @@ public CommandLineApplication(bool throwOnUnexpectedArg = true)
         public Func<string> LongVersionGetter { get; set; }
         public Func<string> ShortVersionGetter { get; set; }
         public readonly List<CommandLineApplication> Commands;
+        public TextWriter Out { get; set; } = Console.Out;
```

Comment:
All the other switches on this are public, including fields like the one on the line above. Was keeping it consistent.